### PR TITLE
import: Mkdir if not exist

### PR DESCRIPTION
**Pull request by ghost**
So something like this?
Related to #33

### CODE REVIEW

File: import/import.go [hunk 1]
```diff
@@ -145,9 +145,14 @@ func doImport(f io.Reader, size int, addTags string,
 // write or something, overwrite it.
 func writeFile(path string, buf []byte) (err error) {
 	const flags = os.O_WRONLY | os.O_CREATE | os.O_TRUNC
+openFile:
 	f, err := os.OpenFile(path, flags, 0660)
 	if err != nil {
-		return
+		if os.MkdirAll(path, 0660) != nil {
```
Comment (bakape) [inline import/import.go]: `path` is the file path. You will need to strip the last segment.
`~/.hydron/images/aa/aa43464554544535.jpg` -> `~/.hydron/images/aa/`

**Review by bakape — CHANGES_REQUESTED** [review]

File: import/import.go (hunk 1 above)
Comment (ghost) [reply import/import.go]: Okay, is there anything else that needs to be done?
I'm kind of walking in the dark.

**Comment by ghost** [comment]
How about now?

File: import/import.go [hunk 2]
```diff
@@ -162,9 +163,14 @@ process:
 // write or something, overwrite it.
 func writeFile(path string, buf []byte) (err error) {
 	const flags = os.O_WRONLY | os.O_CREATE | os.O_TRUNC
+openFile:
 	f, err := os.OpenFile(path, flags, 0660)
 	if err != nil {
-		return
+		if os.MkdirAll(filepath.Dir(path), 0660) != nil {
```
Comment (bakape) [inline import/import.go]: Error from `os.MkdirAll()` is not being returned.

**Review by bakape — CHANGES_REQUESTED** [review]

**Review by bakape — APPROVED** [review]

**Comment by ghost** [comment]
oyasumi lat

**Pull request merged**